app/views/finance/participants/_ecf_profile.html.erb

```diff
@@ -1,57 +1,70 @@
+<h3><%= pp.type %></h3>
+
 <%= govuk_summary_list do |summary_list| %>
   <% summary_list.row do |row| %>
     <% row.key { "User ID / Participant ID" } %>
     <% row.value { @user.id } %>
+    <% row.action %>
   <% end %>
 
   <% summary_list.row do |row| %>
     <% row.key { "Profile ID" } %>
     <% row.value { pp.id } %>
+    <% row.action %>
   <% end %>
 
   <% summary_list.row do |row| %>
     <% row.key { "Eligible for funding" } %>
     <% row.value { pp.fundable?.to_s.upcase } %>
+    <% row.action %>
   <% end %>
 
   <% summary_list.row do |row| %>
     <% row.key { "Lead provider" } %>
     <% row.value { pp&.school_cohort&.school&.active_partnerships[0]&.lead_provider&.name } %>
+    <% row.action %>
   <% end %>
 
   <% summary_list.row do |row| %>
     <% row.key { "School URN" } %>
     <% row.value { pp&.school_cohort&.school&.urn } %>
+    <% row.action %>
   <% end %>
 
   <% summary_list.row do |row| %>
     <% row.key { "Status" } %>
     <% row.value { pp.status } %>
+    <% row.action %>
   <% end %>
 
   <% summary_list.row do |row| %>
     <% row.key { "Training status" } %>
     <% row.value { pp.training_status } %>
+    <% row.action text: "Change", visually_hidden_text: "Change training status", href: edit_finance_participant_profile_path(@user, pp) %>
   <% end %>
```

Comment:
You could put this and the other text in en.yml but I can see this isn't being done already in these views